src/__tests__/App.test.tsx

```diff
@@ -1,11 +1,19 @@
 import React from "react";
-import { render, screen } from "@testing-library/react-native";
-import { describe, expect, it } from "@jest/globals";
+import { render } from "@testing-library/react-native";
 import App from "../App";
+import { useIsShake } from "../useIsShake";
+
+jest.mock("../useIsShake");
+const mockUseIsShake = jest.mocked(useIsShake);
 
 describe("App component", () => {
   it("renders App to screen", () => {
-    render(<App />);
-    expect(screen);
+    mockUseIsShake.mockReturnValue({
```

Comment:
I'd name this test something else than `renders App to screen` -- probably something like `renders AnimateQuotes and AnimateSvg when isShakeTriggered is false`